test/torchaudio_unittest/functional/functional_impl.py

```diff
@@ -582,6 +583,15 @@ def test_rnnt_loss_costs_and_gradients_random_data_with_numpy_fp32(self):
             ref_costs, ref_gradients = rnnt_utils.compute_with_numpy_transducer(data=data)
             self._test_costs_and_gradients(data=data, ref_costs=ref_costs, ref_gradients=ref_gradients)
 
+    def test_rtf_evd(self):
```

Comment:
Please add a dosctring about what is the intended tests. So that we have later clues later, in some unexpected investigation of the code.